ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -1147,8 +1149,18 @@ static MockReplicationManager getReplicationManager(VerifiableProperties verifia
     MockReplicationManager(ReplicationConfig replicationConfig, ClusterMapConfig clusterMapConfig,
         StoreConfig storeConfig, StorageManager storageManager, ClusterMap clusterMap, DataNodeId dataNodeId)
         throws ReplicationException {
-      super(replicationConfig, clusterMapConfig, storeConfig, storageManager, stream -> null, clusterMap, null,
-          dataNodeId, null, clusterMap.getMetricRegistry(), null);
+      super(replicationConfig, clusterMapConfig, storeConfig, storageManager, new StoreKeyFactory() {
```

Comment:
this change needed? `null` won't work anymore?

Reply:
yes, becuase `stream -> null` is a lambda for StoreKeyFactory which no longer works since it has more than one method now